app/assets/stylesheets/application.css.scss

```diff
@@ -44,7 +50,22 @@ body[data-action="devise/sessions-new"], body[data-action="devise/sessions-creat
 }
 
 #footer {
-  font-size: 10px;
+  height: 20px;
+  position: absolute;
+  left: 0;
+  bottom: 0;
+  width: 100%;
+  background-color: #080808;
```

Comment:
Color literals like `#080808` should only be used in variable declarations; they should be referred to via variable everywhere else.